src/client/app/containers/UIOptionsContainer.js

```diff
@@ -23,7 +23,10 @@ function mapStateToProps(state) {
 function mapDispatchToProps(dispatch) {
 	return {
 		selectMeters: newSelectedMeterIDs => dispatch(changeSelectedMeters(newSelectedMeterIDs)),
-		fetchMetersDataIfNeeded: () => dispatch(fetchMetersDataIfNeeded())
+		changeDuration: barDuration => dispatch(changeBarDuration(barDuration)),
+		fetchMetersDetailsIfNeeded: () => dispatch(fetchMetersDetailsIfNeeded()),
+		changeChartType: chartType => dispatch(changeChartToRender(chartType)),
+		changeBarStacking: () => dispatch(changeBarStacking())
```

Comment:
I think changeBarStacking should be called toggleBarStacking since it's actually toggling something.